sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -65,3 +65,12 @@ export function getPartTimeMaritalStatusCode(
   }
   return maritalStatus === RelationshipStatus.Single ? "SI" : "SP";
 }
+
+/**
+ * Get the e-Cert flag for the borrower's persistent or prolonged disability status.
+ * @param hasPDorPPD indicates if a borrower has a persistent or prolonged disability.
+ * @returns "Y" or "N" flag.
+ */
+export function getPPDFlag(hasPPD?: boolean): string {
```

Comment:
If hasPPD is not intended to be an optional field, we can ignore `?`. Just my opinion, feel free to ignore.

Reply:
It is mandatory for the file but optional on SIMS, that is why it receives the optional parameter and converts it to a value.